assets/stylesheets/common/leaderboard-admin.scss

```diff
@@ -0,0 +1,23 @@
+.leaderboard-admin {
+  &__title {
```

Comment:
This `.leaderboard-admin__title` element is an H3, and core has default styles for headings in admin screens, I don't think plugins should be overriding them.

Reply:
True enough. It should be an h1 though actually.

Reply:
Yeah, h1 would technically be the correct answer, except that other screens in the same route also use h3 (Core's own `/admin/plugins`, chat).

Reply:
Okay, that's part of a larger issue then. We should think about starting to fix these html/accessibility mistakes then gradually.